src/index.html

```diff
@@ -3,10 +3,372 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home">
+        <img 
+          class="logo"
+          src="./images/logo.png"
+          alt="Moyologo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="#laptops & computers"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li>
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+          <li>
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <div class="card__cards">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="card__image monitor Apple"
```

Comment:
```suggestion
            alt="Apple monitor"
```
https://www.w3schools.com/tags/att_img_alt.asp